### DISCUSSION TITLE
Mirror traffic to Security Onion

### DISCUSSION THREAD

**Discussion by nicholaswkc34** [Security-Onion-Solutions/securityonion#8285] [General] [answered]
Dear all, I'm planned to install Security Onion 2.3 but i wonder how to mirrow all traffic from WAN to Security onion. 
My Network setup as below: 
Modem -- OPNSense -- Linskys Router as Bridge  --- LAN.

Please help me on this. Thanks.

**Comment by erwintwr2** [comment, marked as answer]
HI Peter

you will need hardware that has mirror capability (typically a network switch)
see bottom of this page :
https://github.com/security-onion-solutions/security-onion/wiki/Hardware

**Reply by dougburks** [reply]
> https://github.com/security-onion-solutions/security-onion/wiki/Hardware

That's an old version of the documentation. Here's the current version:
https://docs.securityonion.net/en/2.3/hardware.html#packets
